public/app/views/shared/_record_innards.html.erb

```diff
@@ -81,7 +81,9 @@
                                                            :p_id => 'classifications_list', :p_body => x} %>
   <% end %>
```

Comment:
It looks like the indentation of the block below somehow got a little wonky (even before this commit).